delft/eris/network.nix

```diff
@@ -0,0 +1,36 @@
+{
+  services.resolved = {
+    enable = true;
+    fallbackDns = [
+      "213.133.98.98"
+      "213.133.99.99"
+      "213.133.100.100"
+      "2a01:4f8:0:a0a1::add:1010"
+      "2a01:4f8:0:a102::add:9999"
+      "2a01:4f8:0:a111::add:9898"
+    ];
+  };
+
+  systemd.network = {
+    enable = true;
+    networks = {
+      "30-enp0s31f6" = {
+        matchConfig = {
+          MACAddress = "90:1b:0e:91:c3:67";
+          Type = "ether";
+        };
+        linkConfig.RequiredForOnline = true;
+        networkConfig.Description = "WAN";
+        address = [
+          "138.201.32.77/26"
+          "2a01:4f8:171:33cc::1/64"
+        ];
+        routes = [ {
+          routeConfig.Gateway = "138.201.32.65";
+        } {
+          routeConfig.Gateway = "fe80::1";
```

Comment:
FWIW on my Hetzner machines I used to need GatewayOnLink=true for this to work. Not sure if that's still the case, but worth keeping in mind if we run into issues with v6 connectivity.

Reply:
We'll have link local addresses, so it will he in the subnet, no onlink flag required in that case. 

https://www.freedesktop.org/software/systemd/man/latest/systemd.network.html#LinkLocalAddressing=